src/main/java/uk/gov/hmcts/reform/roleassignment/util/AuditLoggerUtil.java

```diff
@@ -113,4 +113,17 @@ public static String sizeOfAssignments(final ResponseEntity<RoleAssignmentResour
         }
         return null;
     }
+    public static String responseTime(final  ResponseEntity<RoleAssignmentResource> response) {
```

Comment:
this is not require. you can calculate responsetime directly in AuditAspect.java